src/ploneintranet/userprofile/profiles/default/workflows/userprofile_workflow/definition.xml

```diff
@@ -0,0 +1,190 @@
+    <permission-role>Site Administrator</permission-role>
+   </permission-map>
+  </state>
+
+  <state state_id="enabled" title="Enabled">
+   <exit-transition transition_id="disable" />
+   <permission-map name="View" acquired="False">
+    <permission-role>Authenticated</permission-role>
+    <permission-role>Reader</permission-role>
+    <permission-role>Member</permission-role>
+    <permission-role>Manager</permission-role>
+    <permission-role>Site Administrator</permission-role>
+   </permission-map>
+   <permission-map name="Access contents information" acquired="False">
+    <permission-role>Authenticated</permission-role>
+    <permission-role>Reader</permission-role>
+    <permission-role>Member</permission-role>
+    <permission-role>Manager</permission-role>
+    <permission-role>Site Administrator</permission-role>
+   </permission-map>
+   <permission-map name="Modify portal content" acquired="False">
+    <permission-role>Owner</permission-role>
+    <permission-role>Editor</permission-role>
+    <permission-role>Manager</permission-role>
+    <permission-role>Site Administrator</permission-role>
+   </permission-map>
+   <permission-map name="Change portal events" acquired="False">
+    <permission-role>Reviewer</permission-role>
+    <permission-role>Manager</permission-role>
+    <permission-role>Site Administrator</permission-role>
+   </permission-map>
+  </state>
+
+  <state state_id="disabled" title="Disabled">
+   <exit-transition transition_id="enable" />
+   <permission-map name="View" acquired="False">
+    <permission-role>Reader</permission-role>
+    <permission-role>Reviewer</permission-role>
+    <permission-role>Manager</permission-role>
+    <permission-role>Site Administrator</permission-role>
+   </permission-map>
+   <permission-map name="Access contents information" acquired="False">
+    <permission-role>Reader</permission-role>
+    <permission-role>Reviewer</permission-role>
+    <permission-role>Manager</permission-role>
+    <permission-role>Site Administrator</permission-role>
+   </permission-map>
+   <permission-map name="Modify portal content" acquired="False">
+    <permission-role>Editor</permission-role>
+    <permission-role>Manager</permission-role>
+    <permission-role>Site Administrator</permission-role>
+   </permission-map>
+   <permission-map name="Change portal events" acquired="False">
+    <permission-role>Reviewer</permission-role>
+    <permission-role>Manager</permission-role>
+    <permission-role>Site Administrator</permission-role>
+   </permission-map>
+  </state>
+
+
+  <transition transition_id="approve" new_state="enabled"
+               title="Approve this member"
+               trigger="USER"
+               before_script="" after_script="">
+   <action url="%(content_url)s/content_status_modify?workflow_action=approve"
+           category="workflow">Approve This Member</action>
+   <guard>
+    <guard-permission>Review portal content</guard-permission>
+   </guard>
```

Comment:
Doesn't this take treating members as content too far? I know this is the canonical membrane way, but the security implications of publishing a page are different from those of enabling a user account. Would like some discussions on this.